### PR TITLE
More typings 2

### PR DESCRIPTION
**Pull request by miherlosev**
@LavrovArtem

### CODE REVIEW

**Comment by miherlosev** [comment]
@LavrovArtem FPR

File: src/client/sandbox/event/active-window-tracker.ts [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
-    constructor (messageSandbox) {
+    constructor (private readonly _messageSandbox: MessageSandbox) { //eslint-disable-line no-unused-vars
         super();
 
-        this.messageSandbox = messageSandbox;
-
-        this.isIframeWindow = null;
-        this.activeWindow   = null;
-        this.isActive       = null;
+        this._isIframeWindow = false;
```
Comment (LavrovArtem) [inline src/client/sandbox/event/active-window-tracker.ts]: Let's use the `private` modifier there and in other places if a class property starts with the underscore symbol.

File: src/client/sandbox/event/hover.ts [hunk 2]
```diff
@@ -4,20 +4,19 @@ import nativeMethods from '../native-methods';
...
 import * as positionUtils from '../../utils/position';
 import * as browserUtils from '../../utils/browser';
+/*eslint-disable no-unused-vars*/
+import Listeners from './listeners';
+/*eslint-enable no-unused-vars*/
 
 export default class HoverSandbox extends SandboxBase {
-    listeners: any;
+    _hoverElementFixed: boolean;
+    _lastHoveredElement: any;
```
Comment (LavrovArtem) [inline src/client/sandbox/event/hover.ts]:
```ts
    private _hoverElementFixed: boolean = false;
    private _lastHoveredElement: any = null;
```

File: src/client/sandbox/event/hover.ts [hunk 3]
```diff
@@ -4,20 +4,19 @@ import nativeMethods from '../native-methods';
...
-    constructor (listeners) {
+    constructor (private readonly _listeners: Listeners) { //eslint-disable-line no-unused-vars
         super();
 
-        this.listeners = listeners;
-
-        this.hoverElementFixed  = false;
-        this.lastHoveredElement = null;
+        this._hoverElementFixed  = false;
+        this._lastHoveredElement = null;
```
Comment (LavrovArtem) [inline src/client/sandbox/event/hover.ts]: remove

**Comment by miherlosev** [comment]
FPR

**Pull request merged**